doc/api/http.md

```diff
@@ -1528,7 +1529,20 @@ added to the [`'request'`][] event.
 added: v0.3.6
 -->
 
-* `options` {Object}
+* `options` {Object | string}
+  - `protocol` {string} Defaults to `http:`.
```

Comment:
Elsewhere in the change you using the pattern `Default = ...`. 

Would you please update for consistency? Thank you! 😄